crates/humanode-peer/src/cli/run.rs

```diff
@@ -233,7 +233,7 @@ pub async fn run() -> sc_cli::Result<()> {
             let runner = root.create_humanode_runner(cmd)?;
             runner.sync_run(|config| {
                 let partial = service::new_partial(&config)?;
-                let frontier_backend = match partial.other.4 {
+                let frontier_backend = match partial.other.5 {
```

Comment:
We should unpack the `partial.other` to be more expressive.

Reply:
https://github.com/humanode-network/humanode/pull/861/commits/6694353680fde8312e12144c7dede29b8d0063b9